components/post_view/post_body/post_body.jsx

```diff
@@ -212,6 +222,7 @@ export default class PostBody extends React.PureComponent {
                 <div
                     id={`${post.id}_message`}
                     className={`post__body ${mentionHighlightClass} ${ephemeralPostClass} ${postClass}`}
+                    onClick={this.handleClick}
```

Comment:
This would cause a potential issue with child component which have an action attached to them. Like image previews/attachments and possibly even with buttons created by plugins?. This would be opening the RHS along with the actions associated with these.

@bradjcoughlin Can you check if we can avoid these?

Reply:
@sudheerDev I've added handling to the component to avoid interference with attachments and actions. The click action is now triggered on the `.post-message__text` div, respecting the logic in `Utils.handleFormattedTextClick`.